src/test/java/BicycleManagerTest.java

```diff
@@ -1,78 +1,53 @@
 import ua.lviv.iot.algo.part1.lab3.Tricycle;
 import ua.lviv.iot.algo.part1.lab3.Unicycle;
 
+import org.junit.Test;
 import static org.junit.Assert.assertEquals;
 
-import java.util.ArrayList;
+import java.util.LinkedList;
 import java.util.List;
-import java.util.stream.Collectors;
 
 @Getter
 public class BicycleManagerTest {
-    final List<AbstractBicycle> bicycleList = new ArrayList<>();
-
-    public void addBicycle(final AbstractBicycle bicycle) {
-        this.bicycleList.add(bicycle);
-    }
+    BicycleManager bicycles = new BicycleManager();
 
     @Test
-    public void testFindAllWithMaxSpeedHigherThan() {
-        List<AbstractBicycle> bicycleList = new ArrayList<>();
-        bicycleList.add(new ElectroBicycle("Samsung", 30, 15, 50, 10));
-        bicycleList.add(new Bicycle("Xiaomi", 40, 39, "non-mountain"));
-        bicycleList.add(new Unicycle("Evil Company", 4, 1, 3, 2));
-        bicycleList.add(new Tricycle("Tri-company", 10, 6, true, 0.8F));
-
-        List<AbstractBicycle> sortedList = bicycleList.stream().
-                filter(bicycle -> bicycle.
-                        getBikeMaxSpeedInMPH() > 28).
-                collect(Collectors.toList());
-        List<AbstractBicycle> expectedList = new ArrayList<>();
-        expectedList.add(new ElectroBicycle("Samsung", 30, 15, 50, 10));
-        expectedList.add(new Bicycle("Xiaomi", 40, 39, "non-mountain"));
-
-        assertEquals(sortedList.size(), expectedList.size());
+    public void testAddBicycle() {
+       for (int i = 0; i<5;i++){
+           bicycles.addBicycle(new Bicycle());
+       }
+        assertEquals(5,bicycles.getBicycleList().size() );
     }
 
     @Test
     public void testFindAllWithCurrentSpeedLowerThan() {
-        List<AbstractBicycle> bicycleList = new ArrayList<>();
-        bicycleList.add(new ElectroBicycle("Samsung", 30, 15, 50, 10));
-        bicycleList.add(new Bicycle("Xiaomi", 40, 39, "non-mountain"));
-        bicycleList.add(new Unicycle("Evil Company", 4, 1, 3, 2));
-        bicycleList.add(new Tricycle("Tri-company", 10, 6, true, 0.8F));
-
-        List<AbstractBicycle> sortedList = bicycleList.stream().
-                filter(bicycle -> bicycle.
-                        getBikeCurrentSpeedInMPH() < 20).
-                collect(Collectors.toList());
-        List<AbstractBicycle> expectedList = new ArrayList<>();
-        expectedList.add(new ElectroBicycle("Samsung", 30, 15, 50, 10));
-        expectedList.add(new Unicycle("Evil Company", 4, 1, 3, 2));
-        expectedList.add(new Tricycle("Tri-company", 10, 6, true, 0.8F));
-        assertEquals(sortedList.size(), expectedList.size());
+        bicycles.addBicycle(new ElectroBicycle("Samsung", 30, 15, 50, 10));
+        bicycles.addBicycle(new Bicycle("Xiaomi", 40, 39, "non-mountain"));
+        bicycles.addBicycle(new Unicycle("Evil Company", 4, 1, 3, 2));
+        bicycles.addBicycle(new Tricycle("Tri-company", 10, 6, true, 0.8F));
```

Comment:
this is your personal choice, but I'd rather write

initialBicycles = [new ElectroBicycle("Samsung", 30, 15, 50, 10), new Bicycle("Xiaomi", 40, 39, "non-mountain"), new Unicycle("Evil Company", 4, 1, 3, 2), new Tricycle("Tri-company", 10, 6, true, 0.8F)]

for (var bicycle : initialBicyles) {
    this.bicycles.addBicycle(bicycle)
}

Reply:
I think that for my situation (I don't create many objects) it will be better